.github/workflows/actions.yml

```diff
@@ -1,5 +1,11 @@
 name: build
-on: [push, pull_request]
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
```

Comment:
This was causing duplicate builds, so I copied this from conbench, where I fixed the problem a while ago

Reply:
Could you pull this change out into a separate PR if we still think we want to do it?